sdk/policyinsights/Microsoft.Azure.Management.PolicyInsights/src/Microsoft.Azure.Management.PolicyInsights.csproj

```diff
@@ -7,11 +7,13 @@
     <PackageId>Microsoft.Azure.Management.PolicyInsights</PackageId>
     <Description>Provides policy insights operations for Microsoft Azure.</Description>
     <AssemblyName>Microsoft.Azure.Management.PolicyInsights</AssemblyName>
-    <Version>3.1.0</Version>
+    <Version>3.2.0</Version>
```

Comment:
Version update also needed here

Reply:
Ah, yes, thanks for catching it. Updated.